### PR TITLE
Remove Tensorflow support

### PR DESCRIPTION
**Pull request by stweil**
Tensorflow was never used because of missing models.

### CODE REVIEW

**Comment by stweil** [comment]
See issue #4349. Note that `unittest` still uses TensorFlow code (partially disabled) for different purposes.

File: unittest/third_party/utf/utfdef.h
Comment (stweil) [inline unittest/third_party/utf/utfdef.h:1]: It looks like this file was already unused before the removal of the TensorFlow code.

**Review by amitdo — APPROVED** [review]

**Comment by amitdo** [comment]
Thank you!

File: unittest/third_party/utf/utfdef.h
Comment (stweil) [reply unittest/third_party/utf/utfdef.h:1]: I was wrong and had to fix the unittest in commit 2a1ce80a42ad5eb1ddb1b770e98b13c1864911d2.

**Pull request merged**